modules/core/src/main/scala/com.snowplowanalytics.iglu/client/resolver/registries/Embedded.scala

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2014-2023 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.iglu.client.resolver.registries
+
+// Java
+import com.snowplowanalytics.iglu.core.{SchemaKey, SchemaVer}
+
+import java.io.{File, InputStream}
+import scala.util.matching.Regex
+
+// Scala
+import scala.io.Source
+import scala.util.control.NonFatal
+
+// Cats
+import cats.effect.Sync
+import cats.effect.implicits._
+import cats.implicits._
+import cats.data.EitherT
+
+// circe
+import io.circe.parser.parse
+import io.circe.Json
+
+import com.snowplowanalytics.iglu.core.SchemaList
```

Comment:
we repeatedly rename this to `SchemaKeyList` because of the confusion, maybe we can rename here until the next core release, I opened https://github.com/snowplow/iglu-scala-core/issues/100 to be included in the next core release